src/App.js

```diff
@@ -25,6 +26,7 @@ function App() {
         <Route path="/dashboard/navigator" element={<NavigatorDashboard />} />
         <Route path="/reset" element={<Reset />} />
         <Route path="/profile" element={<Profile />} />
+        <Route path="/navigatordashboard" element={<NavigatorDashboard />} />
```

Comment:
lets remove this route since we already have a route to the navigator dashboard on line 26